open-craft/sprints
# [BB-1244] Sprint planning dashboard

This creates the sprint planning dashboard.

## Testing instructions:
1. Set `JIRA_SERVER`, `JIRA_USERNAME` and `JIRA_PASSWORD` in your envs - `.env`.
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Create `frontend/.env.local` and set `REACT_APP_GOOGLE_CLIENT_ID` there.
1. Navigate to `frontend` and run `npm i && npm start`. Check how the dashboard is working in a newly opened window.
